index.js

```diff
@@ -130,62 +127,64 @@ exports.socketio = function (hook_name, args, cb){
   return cb();
 };
 
-exports.eejsBlock_dd_insert = function (hook_name, args, cb) {
-  args.content = args.content + eejs.require("ep_comments_page/templates/menuButtons.ejs");
+exports.eejsBlock_dd_insert = (hookName, args, cb) => {
+  args.content += eejs.require('ep_comments_page/templates/menuButtons.ejs');
   return cb();
 };
 
-exports.eejsBlock_mySettings = function (hook_name, args, cb) {
-  args.content = args.content + eejs.require("ep_comments_page/templates/settings.ejs");
+exports.eejsBlock_mySettings = (hookName, args, cb) => {
+  args.content += eejs.require('ep_comments_page/templates/settings.ejs');
   return cb();
 };
 
-exports.padInitToolbar = function (hookName, args) {
-  var toolbar = args.toolbar;
+exports.padInitToolbar = (hookName, args) => {
+  const toolbar = args.toolbar;
```

Comment:
just a check to ensure toolbar isn't mutated when ``registerButton`` is fired.

Reply:
`const` only prevents reassignment of that variable to another value—it doesn't mean that the object is immutable.